db/install.xml

```diff
@@ -54,7 +54,7 @@
         <FIELD NAME="guestlinkid" TYPE="char" LENGTH="255" NOTNULL="false" SEQUENCE="false"/>
         <FIELD NAME="guestpass" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false"/>
         <FIELD NAME="moderatorapproval" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
-        <FIELD NAME="guestlinkexpiresat" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="guestlinkexpiresat" TYPE="int" LENGTH="10" NOTNULL="false" DEFAULT="null" SEQUENCE="false"/>
```

Comment:
For when the record is inserted but the site settings does not indicate a default value. It will default to null. When guestlink is then enabled, without expiry being mandatory, but with a default value available, it will assume the default date. But if it has been saved before as unchecked (resolves to zero), it will stay as unchecked until changed by the 'guestlink duration required' setting with a default available